### DISCUSSION TITLE
Unbrick orbi router

### DISCUSSION THREAD

**Discussion by MarinusPretorius** [community/community#70098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I follow firmware upload using mac and blind

Its been running for 7 hours.

Just get << DATA(NUMBER)

Firmware file is 42mb and number in the print is now at

<< DATA(43895)

HOW LONG MORE WOULD IT TAKE

**Comment by Akash1134** [comment]
Hi USERNAME,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
